Sources/DataConnect.swift

```diff
@@ -106,6 +113,18 @@ public class DataConnect {
       callerSDKType: self.callerSDKType
     )
     operationsManager = OperationsManager(grpcClient: grpcClient)
+
+    LoadArguments()
+  }
+
+  private func LoadArguments() {
+    let arguments = ProcessInfo.processInfo.arguments
+    if arguments.contains(kFIRPrivateLogDisabledArgument) {
+      ArgumentFlag.privateLoggingEnabled = false
+      DataConnectLogger.debug("DataConnect private logging disabled.")
+    } else {
+      DataConnectLogger.debug("DataConnect private logging enabled.")
+    }
```

Comment:
related to my other comment, this block could be removed for the suggestion in the `DataConnectLogger.swift` file
```suggestion

    LoadArguments()
  }

  private func LoadArguments() {
    let arguments = ProcessInfo.processInfo.arguments
    if arguments.contains(kFIRPrivateLogDisabledArgument) {
      ArgumentFlag.privateLoggingEnabled = false
      DataConnectLogger.debug("DataConnect private logging disabled.")
    } else {
      DataConnectLogger.debug("DataConnect private logging enabled.")
    }
```